### ISSUE TITLE
Hungarian translation

### ISSUE THREAD

**Issue by Belaba-hub** [luandersonn/TorrentClient-feedback#33]
Hi, if the development is still going on I would like to help with the Hungarian (hu-HU) translation. Please create the language file for Hungarian, or let me now if you gave up this project (I'd be really sad).

**Comment by luandersonn** [comment]
Hi! Thanks a lot for your interest in helping with the Hungarian (hu-HU) translation — I really appreciate it!

I'm happy to let you know that the project is active again, and the branch [`lang/hu-HU`](https://github.com/luandersonn/TorrentClient-feedback/tree/lang/hu-HU) has just been created for you to start working on the translation.

The translation is managed through `.xlf` files, which can be edited using [Poedit](https://poedit.net/) or any other XLIFF-compatible tool of your choice.

Feel free to open a PR to that branch when you're ready. Let me know if you need any help getting started. Thanks again for contributing!

**Comment by luandersonn** [comment]
The file is ready for translation

**Comment by Belaba-hub** [comment]
Yeah, I know, and I already uploaded the translation xlf file more than a year ago in pull request #36, you merged it two weeks ago, then reverted it. I can't follow what is happening here :)

**Comment by luandersonn** [comment]
You're right, it seems I messed up, but I was able to recover the translated file. Thanks for your help. 

faf5c205ec57a84db9b9a929c1ab149c78c282d4 

HU-hu is available on June Update (`>= 1.1.10`)
